### PR TITLE
refactor(admin): remove admin application

### LINKED ISSUE #1785: 🔧 [Refactor] Delete admin application
## 🔧 Refactor

There is no good reason why we deploy the admin interface as separate application from the `frontend/` but there is a number of reason why we should make it part of it:

* Less dependencies and dependabot PRs
* Less workflows to run
* Less code duplication
* Less memory consumption on our target host
* Less domains to generate certificates for
* Less potential for CORS issues
* Less containers running in `docker-compose` and smaller disk usage on our laptops

### PR DESCRIPTION
**Pull request by roschaefer**
Motivation
----------
If we implement role-based authorization, we will enforce it in the backend and we will teach the frontend how to show a separate interface to users with a role `admin`.

This PR will improve the following:

* Less dependencies + dependabot PRs
* Less workflows to run
* Less code duplication
* Less memory consumption in our kubernetes cluster
* Less domains to generate certificates for
* Less potential for CORS issues
* Less containers running in `docker-compose`
* Less disk usage on our laptops

How to test
-----------
1. After this gets merged, we should test our vServer deployment

close #1785

### CODE REVIEW

File: frontend/src/components/menu/UserDropdown.vue [hunk 1]
```diff
@@ -5,37 +5,24 @@
...
 import AuthService from '#src/services/AuthService'
-import { useAuthStore } from '#stores/authStore'
 
 const authService = inject<AuthService>('authService')
 
-const auth = useAuthStore()
-
 async function signOut() {
   try {
-    await authService?.signOut()
```
Comment (roschaefer) [inline frontend/src/components/menu/UserDropdown.vue:30]: Unrelated but I guess OK to get merged along, or shall I open another PR? @Bettelstab

File: frontend/src/layouts/DefaultLayout.test.ts [hunk 2]
```diff
@@ -226,73 +225,4 @@ describe('DefaultLayout', () => {
...
-            AUTH.ADMIN_REDIRECT_URI = 'https://url-to-admin.com'
-            authStore.save({
-              access_token: 'access_token',
-              profile: {
-                aud: 'aud',
-                sub: 'sub',
-                exp: 1,
-                iat: 1,
-                iss: 'iss',
-                groups: ['ADMIN_GROUP'],
```
Comment (roschaefer) [inline frontend/src/layouts/DefaultLayout.test.ts:271]: Authentik documents `group_attributes`. These `group_attributes` is a deep-merge of user attributes as well as group attributes. If a user has these attributes:
```js
{
  "roles": ["moderator"]
}
```
and the user is part of a group called "Dreammall admins" with this:
```js
{
  "roles": ["admin"]
}
```
then the deep-merged `.group_attributes` look like this:
```js
{
  "roles": ["moderator", "admin"]
}
```

That's why I believe this is the right way to implement roles. Not by group name. Authentik also knows the concept of "Roles" but there is no documented way how to get these encoded in the JWT payload (the claims). See this discussion: https://github.com/goauthentik/authentik/discussions/8839

File: deployment/README.md [hunk 3]
```diff
@@ -75,10 +75,6 @@ vi /etc/nginx/http.d/default.conf
 cp deployment/nginx/frontend.conf /etc/nginx/http.d/frontend.conf
 # adjust the frontend nginx config accordingly
 vi /etc/nginx/http.d/frontend.conf
-# admin nginx config
-cp deployment/nginx/admin.conf /etc/nginx/http.d/admin.conf
```
Comment (roschaefer) [inline deployment/README.md:79]: This must be removed manually on our vServer after merge.

File: frontend/src/components/menu/UserDropdown.vue (hunk 1 above)
Comment (roschaefer) [reply frontend/src/components/menu/UserDropdown.vue:30]: OK, wow, this let's a unit test fail. :astonished: I will undo this.

**Review by Elweyn — APPROVED** [review]

**Pull request merged**